WindowsServerDocs/remote/remote-desktop-services/rds-vdi-recommendations-2004.md

```diff
@@ -0,0 +1,794 @@
+    Version      : 13.0.10349.0  
+    Architecture : neutral
+    ResourceId   : \~ 
+    PackageName  : Microsoft.3DBuilder_13.0.10349.0_neutral_\~_8wekyb3d8bbwe
+    Regions      :
+
+    DisplayName  : Microsoft.Appconnector
+    Version      : 2015.707.550  
+    Architecture : neutral
+    ResourceId   : \~ 
+    PackageName  : Microsoft.Appconnector_2015.707.550.0_neutral_\~_8wekyb3d8bbwe
+    Regions      :
+    ...
+```
+UWP apps that are provisioned to a system can be removed during OS installation as part of a task sequence, or later after the OS is installed. This may be the preferred method because it makes the overall process of creating and/or maintaining an image modular. Once you develop the scripts, if something changes in a subsequent build you edit an existing script rather than repeat the process from scratch. Here are some links to information on this topic:
+
+- Microsoft: [Removing Windows 10 in-box apps during a task sequence](https://blogs.technet.microsoft.com/mniehaus/2015/11/11/removing-windows-10-in-box-apps-during-a-task-sequence/)
+- Microsoft: [Removing Built-in apps from Windows 10 WIM-File with PowerShell - Version 1.3](https://gallery.technet.microsoft.com/Removing-Built-in-apps-65dc387b)
+- Microsoft: [Windows 10 1607: Keeping apps from coming back when deploying the feature update](https://blogs.technet.microsoft.com/mniehaus/2016/08/23/windows-10-1607-keeping-apps-from-coming-back-when-deploying-the-feature-update/)
+- Microsoft: [Removing Windows 10 in-box apps during a task sequence](https://blogs.technet.microsoft.com/mniehaus/2015/11/11/removing-windows-10-in-box-apps-during-a-task-sequence/)
+
+Then run the Remove-AppxProvisionedPackage PowerShell command to remove UWP app payloads is as follows:
+
+```powershell
+Remove-AppxProvisionedPackage -Online - PackageName MyAppxPackage 
+```
+
+As a final note on this topic, each UWP app should be evaluated for applicability in each unique environment. You will want to install a default installation of Windows 10 2004, then note which apps are running and consuming memory. For example, you may want to consider removing apps that start automatically, or apps that automatically display information on the Start Menu, such as Weather and News, and that may not be of use in your environment.
+
+> [!NOTE] 
+> If utilizing the scripts from GitHub, you can easily control which apps are removed before running the script. After downloading the script files, locate the file ‘AppxPackage.json’, edit that file, and remove entries for apps that you want to keep, such as Calculator, Sticky Notes, etc.
+
+
+## Windows Optional Features cleanup
+
+### Managing Optional Features with PowerShell
+
+Microsoft: [https://social.technet.microsoft.com/wiki/contents/articles/39386.windows-10-managing-optional-features-with-powershell.aspx](https://social.technet.microsoft.com/wiki/contents/articles/39386.windows-10-managing-optional-features-with-powershell.aspx)
```

Comment:
The recommended codestyle for link text usage is to use either the page title or the most common name for the topic or subject matter at hand in the text part. URLs will usually become links, unmodified, both  on GitHub and on docs.microsoft.com .
```suggestion
Microsoft: [Windows 10: Managing Optional Features with PowerShell](https://social.technet.microsoft.com/wiki/contents/articles/39386.windows-10-managing-optional-features-with-powershell.aspx)
```

Reply:
Fixed